test/mlir/onnx/onnx_canonicalization.mlir

```diff
@@ -96,78 +96,50 @@ func.func @cast_elimination(%arg0: tensor<2xf32>) -> tensor<2xf32> {
 
 // -----
 
-func.func @test_conv_batchnormtestmode_fusion_nobias(%arg0 : tensor<1x3x224x224xf32>) -> tensor<1x64x112x112xf32> {
+func.func @test_conv_batchnormtestmode_fusion_nobias(%arg0: tensor<1x3x224x224xf32>, %0: tensor<64x3x7x7xf32>, %2: tensor<64xf32>, %3: tensor<64xf32>, %4: tensor<64xf32>, %5: tensor<64xf32>) -> tensor<1x64x112x112xf32> {
```

Comment:
any reason to break with the convention to name function args `%arg1` etc?

same in line 123 below